CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 24.0.3 2024-11-22
```

Comment:
This should be `## x.x.x x-x-x` as the date of the release isn't today and the next version number is unknown. This is auto filled out during our release process.